include/exadg/convection_diffusion/preconditioners/multigrid_preconditioner.cpp

```diff
@@ -50,12 +51,14 @@ MultigridPreconditioner<dim, Number>::initialize(
   PDEOperator const &                         pde_operator,
   MultigridOperatorType const &               mg_operator_type,
   bool const                                  mesh_is_moving,
+  Number const                                mg_diffusivity,
```

Comment:
I think one should ask the `pde_operator` for the diffusivity instead of extending the interface.